Core/Registry/Registry.cs

```diff
@@ -231,6 +234,12 @@ private void DeSerialisationFixes(StreamingContext context)
                     ReindexInstalled();
                 }
             }
+            else if (Platform.IsWindows)
+            {
+                // We need case insensitive path matching on Windows
+                // (already done when replacing this object in the above blocks, hence the 'else')
+                installed_files = new Dictionary<string, string>(installed_files, StringComparer.OrdinalIgnoreCase);
+            }
```

Comment:
Shouldn't this be after the `if (registry_version == 0)` block? Like this we're skipping the fix with `registry_version` `1`, no?

Reply:
Ahh yes, I misremembered which block the `StringComparer.OrdinalIgnoreCase` change was in, will move it...

Reply:
No, wait, I had it right originally. The `registry_version < 2` block calls `ReindexInstalled`, which takes care of setting up the case insensitivity. Reverting...